ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/proxy/private/provision/cucushift-installer-rehearse-ibmcloud-ipi-proxy-private-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-proxy-private-provision
```

Comment:
You can directly use `cucushift-installer-rehearse-ibmcloud-ipi-proxy-provision` chain to create a install workflow there, not have to create a new provision chain.

Reply:
Move the env default at cucushift-installer-rehearse-ibmcloud-ipi-proxy-private-workflow ? 
  env:
  - name: RESTRICTED_NETWORK
    default:  "yes"
  - name: PUBLISH
    default:  "Internal"
  - name: BASE_DOMAIN
    default: "ibmcloud.qe.devcluster.openshift.com"

Reply:
In cucushift-installer-rehearse-ibmcloud-ipi-proxy-private-workflow, you only need to add `name: PUBLISH`, others can be added in `cucushift-installer-rehearse-ibmcloud-ipi-proxy-provision`.

Reply:
Do you want to use a different `BASE_DOMAIN` domain name for `cucushift-installer-rehearse-ibmcloud-ipi-proxy-private-workflow`?